NEWS

```diff
@@ -9,6 +39,104 @@ Noteworthy changes in release a.b
 * The old htslib/knetfile.h interfaces have been marked as deprecated.  Any
   code still using them should be updated to use hFILE instead. (#1200)
 
+* Added an introspection API for checking some of the capabilities provided
+  by HTSlib. (#1170) Thanks also to John Marshall for contributions. (#1222)
+    - `hfile_list_schemes`: returns the number of schemes found
+    - `hfile_list_plugins`: returns the number of plugins found
+    - `hfile_has_plugin`: checks if a specific plugin is available
+    - `hts_features`: returns a bit mask with all available features
+    - `hts_test_feature`: test if a feature is available
+    - `hts_feature_string`: return a string summary of enabled features
+
+* Made performance improvements to `probaln_glocal` method, which
+  speeds up mpileup BAQ calculations. (#1188)
+    - Caching of reused loop variables and removal of loop invariants
+    - Code reordering to remove instruction latency.
+    - Other refactoring and tidyups.
+
+* Added a public method for constructing a BAM record from the
+  component pieces. Thanks to Anders Kaplan. (#1159, #1164)
+
+* Added two public methods, `sam_parse_cigar` and `bam_parse_cigar`, as part of
+  a small CIGAR API (#1169, #1182). Thanks to Daniel Cameron for input. (#1147)
+
+
+Build changes
+-------------
+
+These are compiler, configuration and makefile based changes.
+
+* Added new Makefile targets for the applications that embed HTSlib and
+  want to run its test suite or clean its generated artefacts. (#1230, #1238)
+
+* The CRAM codecs are now obtained via the htscodecs submodule, hence
+  when cloning it is now best to use "git clone --recursive".  In an
+  existing clone, you may use "git submodule update --init" to obtain
+  the htscodecs submodule checkout.
+
+* Updated CI test configuration to recurse HTSlib submodules. (#1359)
+
+* Added Cirrus-CI integration as a replacement for Travis, which was
+  phased out.  (#1175; #1212)
+
+* Updated the Windows image used by Appveyor to 'Visual Studio 2019'. (#1172;
+  fixed #1166)
+
+* Fixed a buglet in configure.ac, exposed by the release 2.70 of autoconf.
+  Thanks to John Marshall. (#1198)
+
+* Fixed plugin linking on macOS, to prevent symbol conflict when linking
+  with a static HTSlib. Thanks to John Marshall. (#1184)
+
+* Fixed a clang++9 error in `cram_io.h`. Thanks to Pjotr Prins. (#1190)
+
+* Introduced $(ALL_CPPFLAGS) to allow for more flexibility in setting the
+  compiler flags. Thanks to John Marshall. (#1187)
+
+* Added 'fall through' comments to prevent warnings issued by Clang on
+  intentional fall through case statements, when building with 
+  `-Wextra flag`. Thanks to John Marshall. (#1163)
+
+
+Bug fixes
+---------
+
+* Fixed a bug in the VCF header parser, which prevent it from reading samples
+  names containing UTF-8 characters. Thanks to Sam Morris for reporting the
+  issue. (#1237; fixed samtools/bcftools#1408)
```

Comment:
I don't think _htslib/NEWS_ should state that UTF-8-encoded characters are valid in these sample names, as that is at best unclear in the VCF spec. Suggest describing this fix as something like “Fixed VCF `#CHROM` header parsing to only separate columns at tab characters” — not that that's completely true: actually this hacky code splits at any non-`\n` control character. 🤮  (Also “samples names” typo.)

Reply:
Done. Thanks!